src/util/returnUrlHelper.js

```diff
@@ -4,18 +4,32 @@ const generateReturnUrl = ({ queryParams, isMyAccount }) => {
   const {
     publisherConfig: { adyenConfiguration }
   } = store.getState();
+  let queryParamsString;
 
-  if (isMyAccount && adyenConfiguration?.myaccountReturnUrl)
-    return adyenConfiguration.myaccountReturnUrl;
+  if (isMyAccount && adyenConfiguration?.myaccountReturnUrl) {
+    if (queryParams) {
+      queryParamsString = Object.keys(queryParams).map(
+        key => `${key}=${queryParams[key]}`
+      );
```

Comment:
why we skipped the .join here? could we move this logic to a separate function and simply reuse it?